packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -259,6 +282,25 @@ export class IndexedDbPersistence implements Persistence {
           this.scheduleClientMetadataAndPrimaryLeaseRefreshes()
         );
       })
+      .then(() => {
+        return this.simpleDb.runTransaction(
+          'readonly',
+          [DbTargetGlobal.store],
+          txn => {
+            return getHighestListenSequenceNumber(txn).next(
+              highestListenSequenceNumber => {
+                const sequenceNumberSyncer = this.multiClientParams
```

Comment:
This would just be `this. sequenceNumberSyncer` if you changed the constructor to awlays take a sequence number syncer and provided a NoOp syncer.

... on second thought: What's the side effect of always syncing the sequence numbers? Multi-Tab is optional because it might lead to corruption in IndexedDb if there is a bug/race. If the side effect of always using the synchronization code is that we are writing to LocalStorage, then we could consider always having this enabled.

Reply:
As discussed, let's not always write to local storage.